google/cloud/bigtable_admin_v2/gapic/bigtable_instance_admin_client.py

```diff
@@ -509,43 +510,37 @@ def list_instances(
 
     def update_instance(
         self,
-        name,
         display_name,
-        type_,
-        labels,
+        name=None,
```

Comment:
@crwilcox this is the only other change I noticed that could have the same effect as the one I just fixed. Can you also confirm I didn't miss this/any others?